trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/jdbc/mysql/MySQLDataTypeTransformer.java

```diff
@@ -11,27 +11,62 @@
 import com.datastrato.gravitino.trino.connector.GravitinoErrorCode;
 import com.datastrato.gravitino.trino.connector.util.GeneralDataTypeTransformer;
 import io.trino.spi.TrinoException;
+import io.trino.spi.type.CharType;
 
 /** Type transformer between MySQL and Trino */
 public class MySQLDataTypeTransformer extends GeneralDataTypeTransformer {
   private static final int MYSQL_CHAR_LENGTH_LIMIT = 255;
+  // 65535 / 4 = 16383.
+  private static final int MYSQL_VARCHAR_LENGTH_LIMIT = 16383;
+
```

Comment:
Can you give the reference of this limitation?

Reply:
OK